scripts/analysis.py

```diff
@@ -835,19 +835,20 @@ def plot_power(cursor):
         final_stockpile(cur, 'Separations')
 
         try:
-            pile_dict = get_stockpile(cur, 'Mixer')
-            multi_line_plot(pile_dict, timestep,
+            mixer_stockpile = get_stockpile(cur, 'Mixer')
+            multi_line_plot(mixer_stockpile, timestep,
                             'Years', 'Mass[MTHM]',
                             'Tailings left over in Mixer vs Time',
                             'Total_Stockpile', init_year)
-            pile_dict2 = get_stockpile(cur, 'Separations')
-            multi_line_plot(pile_dict2, timestep,
+        try:
```

Comment:
two try blocks means you also need two except blocks! Anyway, this doesn't really capture the intent of my initial comment. The same (large) amount of code is in try blocks. Now it's just two of them...